product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/pom.xml

```diff
@@ -128,7 +82,7 @@
                             <!-- Directory containing carbon applications -->
                             <name>test.resources.carbonApplications.dir</name>
                             <value>
-                                ${basedir}/../../../../02-SynapseConfigProject/02-synapseConfigCompositeApplication/target/
+                                ${basedir}/../../02-synapseConfigProject/02-synapseConfigCompositeApplication/target/
```

Comment:
Do the paths need to change?

Reply:
Yes since 02-synapseConfigProject and resource are in the same level this patch is correct